scripts/check-manifests.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Copyright 2020 NVIDIA
```

Comment:
s/2020/2021

Reply:
thanks, will fix it once I'll figure out why it doesn't work on CI